examples/cpp/pyperf/PyPerfProfiler.h

```diff
@@ -67,7 +67,8 @@ class PyPerfProfiler {
   // init must be invoked exactly once before invoking profile
   PyPerfResult init(unsigned int symbolsMapSize, unsigned int eventsBufferPages,
                     unsigned int kernelStacksMapSize, unsigned int userStacksPages,
-                    unsigned int fsOffset, unsigned int stackOffset);
+                    unsigned int fsOffset, unsigned int stackOffset,
+                    bool insertDsoName=true);
```

Comment:
Any reason we add a default value for it? The c'tor is invoked only at one site afaik

Reply:
No, no default value needed. Thanks for spotting this.